### DISCUSSION TITLE
Why can't I receive the Ready event without including GatewayIntents.GuildMembers ?

### DISCUSSION THREAD

**Discussion by Wren-ofNetFae** [discord-net/Discord.Net#2822] [Q&A] [answered]
I may be misunderstanding something fundamental about `DiscordSocketClient` so please correct me if so.

My project does not need any protected permissions so I currently use `GatewayIntents = GatewayIntents.AllUnprivileged`.
With this I never get the `DiscordSocketClient.Ready` event.  I do get the `DiscordSocketClient.Connected` event though.
When I just add `GatewayIntents.GuildMembers` to my GatewayIntents then I do get the `DiscordSocketClient.Ready` event.

I have tried both enabling and disabling the `SERVER MEMBERS INTENT` in the Bot in the Developer portal.

I was able to reproduce this in my project as well as in the `InteractionFramework` sample app. v3.13.0
https://github.com/discord-net/Discord.Net/tree/dev/samples/InteractionFramework

Am I misunderstanding something?

**Comment by Misha-133** [comment, marked as answer]
set `AlwaysDownloadUsers` to `false`

**Reply by Wren-ofNetFae** [reply]
Thank you so much. I just confirmed this was my issue.

**Comment by Misha-133** [comment]
For it to work you have to enable the `GatewayIntents.GuildMembers` intent.
DNet has a check for this, it throws an exception when one is not enabled, but before 3.13.0 it was ignored.
BUT. due to some changes in 3.13.0 the exception is now preventing the client from getting to the `Ready` event. 
This issue has been addressed already, it will log a warning instead, we will release a patch soon™
